importScripts/sampleRecords/library.destination.json

```diff
@@ -76,7 +74,158 @@
     {
       "object_id":"3",
       "id":"3",
-      "idno":"101",
+      "display_label":"The myth of equality: the denial of citizenship rights for Aboriginal people in Western Australia (Studies in Western Australian history Vol. 22, 2001, pp.69-82)",
```

Comment:
These are sample values represented by lines 63-68 in `library.sql` below. This particular record had no IDNO.